tests/test_tools_manually.yml

```diff
@@ -44,7 +44,7 @@
     - variant_calling
   files:
     - path: results/csv/variantcalled.csv
-      md5sum: f87290ce1c6ea523e08354ed6c258b0b
```

Comment:
@asp8200 do you want to merge your PR first? You were right, the changed checksums match the joint ones, so i am guessing it could accidentally c&p when I made changes there

Reply:
It seems to me that you also fixed the md5sums in this PR, so how about I just close my PR without merging?

Reply:
sure whatever you prefer. Yeah fixed them because I wanted to check that all the tests pass here

Reply:
I've closed my PR without merging.